website/docs/commands/compile.md

```diff
@@ -209,6 +209,22 @@ There's a separate directive for passing multiple options at one time:
 @main def hello = if (true) println("Hello")
 ```
 
+### Explicit handling of paths in using directives
```

Comment:
I wonder if this section belongs in the `compile` command doc... wouldn't it make more sense in the using directives guide? since it applies to all of them, no?